packages/deployment/app/cloudformation.template.yml

```diff
@@ -453,6 +453,8 @@ Resources:
     {{/each}}
       Handler: {{this.handler}}
       MemorySize: {{this.memory}}
+      TracingConfig:
```

Comment:
Thanks for the contribution! Can we make enabling X-ray conditional on a Lambda config value, like `useXRay` or something? That way, it can be opt-in for anyone who wants it